adafruit_display_text/label.py

```diff
@@ -393,14 +395,17 @@ def anchored_position(self):
 
     @anchored_position.setter
     def anchored_position(self, new_position):
-        new_x = int(
-            new_position[0]
-            - self._anchor_point[0] * (self._boundingbox[2] * self._scale)
-        )
-        new_y = int(
-            new_position[1]
-            - (self._anchor_point[1] * self._boundingbox[3] * self._scale)
-            + round((self._boundingbox[3] * self._scale) / 2.0)
-        )
-        self.x = new_x
-        self.y = new_y
+        if (self._anchor_point is None) or (new_position is None):
+            pass  # Note: anchor_point must be set before setting anchored_position
```

Comment:
```suggestion
            return  # Note: anchor_point must be set before setting anchored_position
```

If you return, then you don't need the else and can leave the existing indentation.